src/CustomerProfile.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+export default function CustomerProfile({ customerProfile }) {
+  return (
+    <div>
+      <h5 class="modal-title">Profile</h5>
+
+      {customerProfile.length !== 0 ? (
+        <ul>
+          <li>Id: {customerProfile.id}</li>
+          <li>Email: {customerProfile.email}</li>
+          <li>Vip: {customerProfile.vip ? "Yes" : "No"}</li>
+          <li>Phone no: {customerProfile.phoneNumber}</li>
+        </ul>
+      ) : (
+        <></>
+      )}
```

Comment:
a ternary operator works here, but you can also use `&&` in react, for example:
```
 {customerProfile.length !== 0 && (
 //code you want to display if the statement is true
```
this is handy if you don't want to render something if the statement is false